tests/requirements.txt

```diff
@@ -1 +1 @@
-dimod==0.12.17
+dimod==0.12.17; python_version < '3.14'
```

Comment:
This isn't causing issues with 3.14 tests?

Reply:
Nope, it's optional